types/react-table/index.d.ts

```diff
@@ -852,6 +852,7 @@ export interface UseSortByColumnProps<D extends object> {
     isSortedDesc: boolean | undefined;
 }
 
+export type SortByFnPartial<D extends object> = (rowA: Row<D>, rowB: Row<D>) => number;
```

Comment:
I think that it would be more consistent to call this `OrderByFn` rather than `SortByFnPartial`. Other than that I think that this is correct.

Reply:
Thanks for the suggestion. Implemented in [e6fa873](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/59100/commits/e6fa873ce9cbb61322f82dbe7c8c51b766bc0303).